packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx

```diff
@@ -261,6 +271,8 @@ class AudioPlayerWithYoutubeSpotify extends Component {
             onSelected={this.selectThisTrack}
             selectedTrack={trackSelected}
             albumName={title[0]}
+            displayTrackNumbers={isArchiveChannel}
+            creator={creator[0]}
```

Comment:
can't creator be `''` above? Does this need to be something like `creator.length ? creator[0] : ''`?

Reply:
the incoming data has been normalized so values come in as arrays.  Thanks for the catch